optaplanner-docs/pom.xml

```diff
@@ -44,6 +44,13 @@
         <plugin>
           <groupId>org.asciidoctor</groupId>
           <artifactId>asciidoctor-maven-plugin</artifactId>
+          <dependencies>
+            <dependency>
+              <groupId>io.netty</groupId>
+              <artifactId>netty-codec-http</artifactId>
+              <version>${version.io.netty}</version>
+            </dependency>
+          </dependencies>
```

Comment:
You can add this dependency in kie-parent instead:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L1324